tests/python/unittest/test_operator.py

```diff
@@ -1906,6 +1906,88 @@ def test_depthwise_convolution():
                             for arr1, arr2 in zip(exe1.outputs + exe1.grad_arrays, exe2.outputs + exe2.grad_arrays):
                                 np.testing.assert_allclose(arr1.asnumpy(), arr2.asnumpy(), rtol=1e-3, atol=1e-3)
 
+
+@with_seed()
+def test_convolution_independent_gradients():
```

Comment:
We have reported flakiness on this UT and added some notes. Seems the the cherry-picked commit is behind the merged one on master branch. And it looks good on master. Would you 
 mind checking it again? Thanks. https://github.com/apache/incubator-mxnet/blob/57927a932610581a351e4aed4fb7c9209d0e57c2/tests/python/unittest/test_operator.py#L2003-L2006

Reply:
Sorry for a mistake. The flakiness is fixed by #15631. I think these two should be taken into v1.5.x together.